console/console-init/ui/src/modules/iot-device/utils/utils.ts

```diff
@@ -184,6 +185,65 @@ const getFormInitialStateByProperty = (
   return initialState;
 };
 
+const getDeviceFromDeviceString = (device: string) => {
```

Comment:
Please change the name of function something like serializeDevice etc.
Improve logic of this function as we had discussed in call. Don't need try catch here.

Reply:
WIll improve this logic in my next PR once your PR get merged to master